modules/admin-ui/src/test/resources/test/unit/modules/events/controllers/serieControllerSpec.js

```diff
@@ -36,6 +36,7 @@ describe('Serie controller', function () {
         $httpBackend.whenGET('/admin-ng/series/4581/participation.json').respond(JSON.stringify(getJSONFixture('admin-ng/series/4581/participation.json')));
         $httpBackend.whenPUT('/admin-ng/series/4581/theme').respond('{}');
         $httpBackend.whenGET('/admin-ng/resources/THEMES.NAME.json').respond({1001: 'Heinz das Pferd', 1002: 'Full Fledged', 401: 'Doc Test'});
+        $httpBackend.whenGET('/admin-ng/resources/THEMES.DESCRIPTION.json').respond({901:"theme1 description",902:"theme2 desc\nsecond line"});
```

Comment:
Should there be spacing in the object literal as in the line above for consistency?